### PR TITLE
Blaka Ink: Version 1.001; ttfautohint (v1.8.4.7-5d5b) added

### PR DESCRIPTION
**Pull request by yanone**
e49f95c: [gftools-packager] Blaka Ink: Version 1.001; ttfautohint (v1.8.4.7-5d5b) added

* Blaka Ink Version 1.001; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/Gue3bara/Blaka at commit https://github.com/Gue3bara/Blaka/commit/8f2ddd99ac2699339a75f7fa76c7e5c7f7a59f8b.

### CODE REVIEW

**Comment by RosaWagner** [comment]
Maybe issue with an external open corner on tildecomb
Capture d’écran 2022-04-21 à 11 20 00

**Comment by yanone** [comment]
Yes. These need to be removed manually because the remove open corner filter fails on the really complicated construction that the designer made here. I thought I'll let it pass here because it's a color font and the b/w is only the fallback. But you're right, it should look good in the fallback, too.
Btw, none of the Browserstack screenshots show the color version. We need to be aware of this when reviewing. Maybe we should add a COLRv1 label?

**Comment by RosaWagner** [comment]
Agreed, maybe the proofing could be improved show color fonts as well ? cc @m4rc1e

**Comment by yanone** [comment]
Updated

## Blaka Ink: Version 1.001; ttfautohint (v1.8.4.7-5d5b) added

---
 5b1255f: [gftools-packager] Blaka Ink: Version 1.001; ttfautohint (v1.8.4.7-5d5b) added

* Blaka Ink Version 1.001; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/Gue3bara/Blaka at commit https://github.com/Gue3bara/Blaka/commit/e6d837a25307db2238371c9b80dd3593b0d98fb3.

**Comment by yanone** [comment]
Fixed it

File: ofl/blakaink/DESCRIPTION.en_us.html [hunk 1]
```diff
@@ -1 +1,18 @@
...
+    to replace a usual baseline in most letterforms, giving the Arabic letterforms contemporary features.
+</p>
+<p>
+    The font family comes in two styles, Blaka and Blaka Hollow, featuring an outlined version of the font. Blaka also
+    comes in two variants, the usual monochrome variants as well as an Ink variant <em>Blaka Ink</em>. The Ink variant
+    are color fonts where the colors are defined by the font itself. Support for color fonts is currently limited to few
+    applications like Google Chrome (version 98 or later).
+</p>
+<p>
+    To contribute, see <a href="https://github.com/Gue3bara/Blaka">github.com/Gue3bara/Blaka</a>.
```
Comment (davelab6) [inline ofl/blakaink/DESCRIPTION.en_us.html:17]: No final period on these lines please :) @felipesanches could be a FB check

**Comment by davelab6** [comment]
@RosaWagner this looks good to me

@rsheeter what was that extra step you wanted to add to the onboarding process? :)

**Comment by rsheeter** [comment]
> @rsheeter what was that extra step you wanted to add to the onboarding process? :)

Replied in more detail in email, tl;dr run https://github.com/googlefonts/nanoemoji#adding-color-tables-to-existing-fonts on incoming color fonts so they have lots of color tables, at least one that works on any given major browser. Brand new tool, file bugs if/when you find them and @anthrotype and I will fix.

File: ofl/blakaink/DESCRIPTION.en_us.html (hunk 1 above)
Comment (RosaWagner) [reply ofl/blakaink/DESCRIPTION.en_us.html:17]: Why?

**Comment by yanone** [comment]
Updated

## Blaka Ink: Version 1.001; ttfautohint (v1.8.4.7-5d5b) added

---
 fb8c318: [gftools-packager] Blaka Ink: Version 1.001; ttfautohint (v1.8.4.7-5d5b) added

* Blaka Ink Version 1.001; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/Gue3bara/Blaka at commit https://github.com/Gue3bara/Blaka/commit/df0ea4f92f29bb6af8dc9984d81fe1aaec1a7186.

**Comment by yanone** [comment]
I found more overlap mistakes (this time in the color version) and uploaded another font binary and fixed the period in the description.

Rod said by email:
> Bonus points for running maximum_color on Blaka Ink, also fine to merge the PR without it.

so please go ahead and review this one more time and then merge it. I tried to reproduce the creation of the missing SVG table as per Rod’s instructions, but failed partially and documented the results [here](https://github.com/googlefonts/nanoemoji/issues/406#issuecomment-1118616973). Rod seems to know what to do, since the google/fonts currently already contains a maximized color fonts which will probably be overwritten by this PR.

**Pull request closed without merging**